rake/site.rake

```diff
@@ -47,7 +47,7 @@ namespace :site do
   task :update_normalize_css do
     Dir.chdir("#{docs_folder}/_sass") do
       sh 'curl "https://necolas.github.io/normalize.css/latest/normalize.css" -o "normalize.scss"'
-      sh 'sass "normalize.scss":"_normalize.scss" --style compressed'
+      sh 'sass "normalize.scss":"_normalize.scss"'
```

Comment:
It seems that previously we used a Sass workflow, and now we rely entirely on GitHub Pages pipeline. We therefore can simplify here to the bare minimum, you're right. 

Suggestion applied. 🖖